src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/StaffRefCreateFunctionalTest.java

```diff
@@ -125,4 +135,142 @@ void createStaffProfile_CwdAdminRoleOnly() {
+                .createStaffProfileCreationRequest();
+        assertNotNull(staffProfileCreationRequest);
+        Response response = caseWorkerApiClient.createStaffUserProfile(staffProfileCreationRequest);
+        assertNotNull(response);
+        StaffProfileCreationResponse staffProfileCreationResponse =
+                response.getBody().as(StaffProfileCreationResponse.class);
+        assertNotNull(staffProfileCreationResponse);
+        assertNotNull(staffProfileCreationResponse.getCaseWorkerId());
+
+        String firstCaseworkerId = staffProfileCreationResponse.getCaseWorkerId();
+
+
+        Response fetchResponse = caseWorkerApiClient.getMultipleAuthHeadersInternal(ROLE_CWD_SYSTEM_USER)
+                .body(UserRequest.builder().userIds(List.of(firstCaseworkerId)).build())
+                .post("/refdata/case-worker/users/fetchUsersById/")
+                .andReturn();
+        fetchResponse.then()
+                .assertThat()
+                .statusCode(200);
+
+        List<uk.gov.hmcts.reform.cwrdapi.client.domain.CaseWorkerProfile> fetchedList =
+                Arrays.asList(fetchResponse.getBody().as(
+                        uk.gov.hmcts.reform.cwrdapi.client.domain.CaseWorkerProfile[].class));
+        assertEquals(1, fetchedList.size());
+        uk.gov.hmcts.reform.cwrdapi.client.domain.CaseWorkerProfile caseWorkerProfile =
+                fetchedList.get(0);
+
+        assertEquals(firstCaseworkerId, caseWorkerProfile.getId());
+        assertEquals(staffProfileCreationRequest.getFirstName(), caseWorkerProfile.getFirstName());
+        assertEquals(staffProfileCreationRequest.getLastName(), caseWorkerProfile.getLastName());
+        assertEquals(staffProfileCreationRequest.getEmailId(), caseWorkerProfile.getOfficialEmail());
+        assertEquals(staffProfileCreationRequest.isStaffAdmin() ? "Y" : "N", caseWorkerProfile.getStaffAdmin());
+        assertEquals(staffProfileCreationRequest.getBaseLocations().size(), caseWorkerProfile.getLocations().size());
+        assertEquals(staffProfileCreationRequest.getBaseLocations().get(0).getLocation(),
+                caseWorkerProfile.getLocations().get(0).getLocationName());
+        assertEquals(staffProfileCreationRequest.getServices().size(), caseWorkerProfile.getWorkAreas().size());
+        assertEquals(staffProfileCreationRequest.getServices().get(0).getService(),
+                caseWorkerProfile.getWorkAreas().get(0).getAreaOfWork());
+        assertEquals(staffProfileCreationRequest.getRoles().size(),
+                caseWorkerProfile.getRoles().size());
+        assertEquals(staffProfileCreationRequest.getRoles().get(0).getRole(),
+                caseWorkerProfile.getRoles().get(0).getRoleName());
+        assertEquals(staffProfileCreationRequest.getServices().size(),
+                caseWorkerProfile.getWorkAreas().size());
+        assertEquals(0L, caseWorkerProfile.getSkills().size());
+        assertThat(caseWorkerProfile.getSkills().isEmpty());
```

Comment:
this assertion is ignored, move closing bracket after getSkills() , like below:
assertThat(caseWorkerProfile.getSkills()).isEmpty();

Reply:
Hi @rajkatla-hmcts . NeedFul Done